src/utils/react.js

```diff
@@ -12,3 +13,21 @@ export let wrapDisplayName = (name, Component) => Wrapped => {
   Wrapped.displayName = `${name}(${getDisplayName(Component)})`
   return Wrapped
 }
+
+// these are for futil :)
+// (x -> y) -> string -> {x} -> y
+let getWith = _.curry((customizer, path, object) =>
+  customizer(_.get(path, object))
+)
+// ({a} -> {b}) -> {a} -> {a, b}
+let expandObject = _.curry((transform, obj) => ({ ...obj, ...transform(obj) }))
+
+// (a -> {b}, k) -> Component<{k: a}> -> Component<{b}>
+export let lensify = _.flow(
+  getWith,
+  expandObject,
+  mapProps
+)
+
+let open = (...lens) => ({ open: F.view(...lens), onClose: F.off(...lens) })
```

Comment:
This is not generic. It's specific to material-ui

Reply:
This follows the same model as the futil domLens functions like `value`, `hover`, and `focus`:

https://github.com/smartprocure/futil-js/blob/master/src/lens.js#L83-L87

But you are right - it doesn't seem generic enough to live in futil itself, which is why it lives here.

Reply:
It's not generic because `open` and `onClose` are material UI props

Reply:
thats fair, we should move open + openify to utils in material 👍